MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1150 @@
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {   
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Date" && e.DataFormItem is DataFormDateItem dateItem)
+        {
+            dateItem.Format = "dd, MM, yyyy";
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+#### Change the minimum and maximum date in date editor
+
+You can customize the maximum and minimum allowable dates in the [DatePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/datepicker?view=net-maui-7.0) by setting `MaximumDate` and `MinimumDate` in the `DataFormDateItem` respectively.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Date" && e.DataFormItem is DataFormDateItem dateItem)
+        {
+            dateItem.MinimumDate = new DateTime(2022, 5, 5);
+            dateItem.MaximumDate = new DateTime(2022, 9, 2);
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Time editor
+
+In the time editor, the [TimePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/timepicker?view=net-maui-7.0&tabs=windows) will be loaded.
+
+In the time editor, the default time value (12:00 AM) is displayed.
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataType(DataType.Time)]
+    [Display(Name = "Event Time")]
+    public TimeSpan? EventTime { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+#### Change the format of time editor**
+
+In the time editor, short time will be shown by default. You can change the applied format by setting the `Format`property in `DataFormTimeItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "EventTime" && e.DataFormItem is DataFormTimeItem timeItem)
+        {
+            timeItem.Format = "HH:mm";
+        }
+    } 
+
+{% endhighlight %}
+{% endtabs %}
+
+## CheckBox editor
+
+In [CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0) editor, the [CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0) control is loaded. By default, for `bool` data type property, the [CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0) editor will be loaded in data form.
```

Comment:
In the CheckBox editor, the [CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0) control is loaded. By default, for `bool` data type property, the CheckBox editor will be loaded in data form.

Reply:
content updated